pkg/webhook/webhook.go

```diff
@@ -571,6 +608,9 @@ func MutateHandler(w http.ResponseWriter, req *http.Request) {
 				}
 			}
 
+			/* patch the pod with additional node selector labels */
+			patch = updateNodeSelector(existingNsMap, desiredNsMap, patch)
```

Comment:
nit: move patch as first argument in updateNodeSelector? so that it's consistent with other patch functions.

Reply:
done